src/tree/registries/UnifiedRegistryTreeDataProvider.ts

```diff
@@ -130,4 +131,32 @@ export class UnifiedRegistryTreeDataProvider implements vscode.TreeDataProvider<
 
         void this.refresh();
     }
+
+    public async getAllConnectedRegistries(): Promise<UnifiedRegistryItem<CommonRegistry>[]> {

```

Comment:
I'm a little concerned how the performance of this could be but let's give it a try and see how it goes.

Reply:
yeah me as well, before I was able to filter my subscriptions (there were 70+, and i filtered it to 1), it took well over 15 seconds to load. But I think there is space for optimization.